### PR TITLE
Feat/Fetch `subnet` SH from NNS

### LINKED ISSUE #1017: Remove `-a` flag from `morph subnet` commands
neofs-adm can fetch address of subnet contract directly from NNS

### PR DESCRIPTION
**Pull request by carpawell**
Also, delete `-a` (`--contract`) flag.

Closes #1017.

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/subnet.go [hunk 1]
```diff
@@ -105,18 +101,19 @@ func initSubnetClientCheckNotary(c *morphsubnet.Client, key *keys.PrivateKey, ch
 		return errors.New("missing endpoint")
 	}
 
-	// read contract address
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/morph/subnet.go:108]: Not useless comment IMO

File: cmd/neofs-adm/internal/modules/morph/subnet.go [hunk 2]
```diff
@@ -105,18 +101,19 @@ func initSubnetClientCheckNotary(c *morphsubnet.Client, key *keys.PrivateKey, ch
...
-		return fmt.Errorf("subnet contract address: %w", err)
-	}
-
 	// create base morph client
 	cMorph, err := client.New(key, endpoint)
 	if err != nil {
 		return err
 	}
 
+	const nnsSubnetKey = "subnet.neofs"
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/morph/subnet.go:110]: `client.NNSSubnetworkContractName `?

Comment (alexvanin) [inline cmd/neofs-adm/internal/modules/morph/subnet.go:110]: https://github.com/nspcc-dev/neofs-node/blob/f9bcb6f5e35ada86eea74e38506600c1837254a3/pkg/morph/client/nns.go#L34

Comment (carpawell) [reply cmd/neofs-adm/internal/modules/morph/subnet.go]: forget that it is a public const

reused it

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**